doc/sphinx/credits.rst

```diff
@@ -1404,3 +1404,137 @@ The contacts of the Coq Consortium are Yves Bertot and Maxime Dénès.
 | Santiago de Chile, March 2018,
 | Matthieu Sozeau for the |Coq| development team
 |
+
+Credits: version 8.9
+--------------------
+
+|Coq| version 8.9 contains the result of refinements and stabilization
+of features and deprecations or removals of deprecated features,
+cleanups of the internals of the system and API along with a few new
+features. This release includes many user-visible changes, including
+deprecations that are documented in ``CHANGES.md``.
+
+TODO:
+We only list the most important ones:
+or
+The main user visible changes are:
+
+- Kernel: mutually defined records are now supported, by P.M. Pédrot.
+
+- Notations: support for autonomous grammars of terms called "custom
+  entries", by H. Herbelin (see chapter "Syntax extensions" of the
+  reference manual).
+
+- Notations: deprecated notations will be removed in the next version of
+  |Coq|, see the``CHANGES.md`` file for a script to ease porting,
+  by J. Gross.
+
+- Tactics: Introduction tactics `intro`/`intros` on a goal that is an
+  existential variable now force a refinement of the goal into a
+  dependent product rather than failing.
+
+- Decision procedures: deprecation of tactics `fourier` and `romega` in
+  favor of `lra` and `lia` respectively.
+
+- Proof language: focusing bracket `{` now supports named goal selectors,
+  e.g. `[x]: {` will focus on a goal (existential variable) named `x`.
+
+- Ltac: Support for `fix`/`cofix` added in Ltac `match` and `lazymatch`.
+
+- SSReflect: delayed clear switches have a simpler semantics and empty
+  clear switches are allowed before view applications of variable views.
+
+- Vernacular:
+
+  - Experimental support for attributes on commands by V. Laporte, as in
+    `#[local] Lemma foo : bar.`
+    Tactics and tactic notations now support the `deprecated` attribute.
+
+  - Removed deprecated commands `Arguments Scope` and `Implicit
+    Arguments` in favor of `Arguments`.
+
+  - New option `Uniform Inductive Parameters` by J. Hugunin to avoid
+    repeating uniform parameters in constructor declarations.
+
+  - New commands `Set Hint Variables/Constants Opaque/Transparent`
+    commands for controlling the opacity flag of variables and constants
+    in hint databases.
+
+  - Added the `Numeral Notation` command for registering decimal numeral
```

Comment:
Perhaps this should be classed with notations?

Reply:
Indeed, I made a grouping of all Notations-related changes